lib/web_ui/dev/test_platform.dart

```diff
@@ -684,12 +684,21 @@ class BrowserManager {
   static Future<BrowserManager> start(
       Runtime runtime, Uri url, Future<WebSocketChannel> future,
       {bool debug = false}) {
+    final Stopwatch stopwatch = new Stopwatch();
+    final int timeoutMiliseconds = 30000;
```

Comment:
30 seconds may not be enough when launching a brand new simulator. Sometimes it takes minutes to launch one. I think we should not have this timeout at all, as timeouts lead to flakiness. The global timeout should be sufficient to handle the case when the whole thing gets stuck.

Reply:
done. thanks for the suggestion